daemon/src/monitor.rs

```diff
@@ -675,7 +640,7 @@ impl xtra::Actor for Actor {
                 }
             },
             |e| async move {
-                tracing::warn!("Failed to re-initialize monitoring: {e:#}");
+                tracing::warn!("Failed to re-broadcast transactions: {e:#}");
```

Comment:
The error became somewhat misleading, as we folded 2 distinct tasks into one:

```suggestion
                tracing::warn!("Failed to re-broadcast transactions or init the monitoring: {e:#}");
```